website/src/content/docs/usage/multiple-cursors.mdx

```diff
@@ -0,0 +1,286 @@
+
+<T.SelectedEntireFileAgain />
+
+Now, with the power of <kbd>s</kbd> and Regex, we can select the `length`, `width` and `height` words by chaining them together with the "OR" operator. So for instance, type the following: `length|width|height`:
+
+<T.SelectedAllLengthWidthHeight />
+
+This is certaintly simpler but it's good to know both approaches. Hit <kbd>Enter</kbd> to select and yet again, go to the beginning of each word with <kbd>b</kbd> and then move 3 characters to the right with <kbd>lll</kbd>, press <kbd>e</kbd> to go to the end of each word and delete it:
+
+<T.RemovedLastPartOfEachParam />
+
+Now we're going to remove the variable declarations of `result` on each line, and instead directly return the calculation.
+
+Go ahead and select the entire file again with <kbd>%</kbd>. And then use the <kbd>s</kbd> to bring up the select prompt again. Type `result =` which will select all instances of `result =`:
+
+<T.SelectedAllResultVars />
+
+<kbd>Enter</kbd> to select, and then Press <kbd>C</kbd> to duplicate the selection on each line:
+
+<T.SelectedAllResultVarsAndC />
+
+By using <kbd>Alt</kbd> + <kbd>(</kbd>, we can rotate the _contents_ of the selection backward:
+
+<T.RotatedContentsOfSelections />
+
+Try using <kbd>Alt</kbd> + <kbd>)</kbd> to rotate the selection forward. You can keep on pressing the key but the difference between <kbd>Alt</kbd> + <kbd>)</kbd> and <kbd>Alt</kbd> + <kbd>(</kbd> won't be obvious in this example, since we basically have the same selections repeated.
+
+Regardless, press <kbd>Alt</kbd> + <kbd>)</kbd> at least once to get into the following state:
+
+<T.RotatedContentsOfSelections />
+
+Collapse each cursor so that there is only a single selection with <kbd>;</kbd>:
+
+<T.EachSelectionCollapsed />
+
+Now, just press <kbd>d</kbd> twice to delete two characters on each of the 6 lines:
+
+<T.DeletedTwoChars />
+
+Okay so at this point, we are _almost_ there. We just need to delete the `result sult` line.
+
+To do this, use the <kbd>s</kbd> again and type `s` in the prompt. This will select the `s` in each of your selections and place a cursor there:
+
+<T.SelectedAllS />
+
+Press <kbd>Enter</kbd> to select, and then press <kbd>x</kbd> to select the entire line and <kbd>d</kbd> to delete it. Voila!
+
+<T.Deleted3Lines />
+
+We've got just a few steps left. Namely, select the entire file again with <kbd>%</kbd>:
+
+<T.SelectedEntireFileAgain2 />
+
+Press <kbd>\></kbd> which will indent everything by 1 level. We need this since we're going to nest it in a class, after all. You can also use <kbd>\<</kbd> to de-dent the selection by 1 level too.
+
+<T.IndentAllBy1Level />
+
+Now, press <kbd>i</kbd> to go into insert mode at the very beginning of the selection, <kbd>Enter</kbd> to add a newline, <kbd>Esc</kbd> to enter normal mode, <kbd>k</kbd> to go to the line above, <kbd>I</kbd> to enter Insert mode and then type `class Calculator:` and <kbd>Esc</kbd> to go back to normal mode.
```

Comment:
Now, press <kbd>i</kbd> to go into insert mode at the very beginning of the selection, <kbd>Enter</kbd> to add a newline, <kbd>Esc</kbd> to enter normal mode, <kbd>k</kbd> to go to the line above, <kbd>I</kbd> to enter 
-> 
Now, press <kbd>O</kbd> to enter a newline above in